controller/StipendRequestController.js

```diff
@@ -0,0 +1,63 @@
+const catchAsyncError = require("../middleware/catchAsyncError");
+const ErrorHandler = require("../utils/ErrorHandler");
+const { StipendRequest, Mail } = require("../services");
+const {
+  validateStipendRequest,
+  stipendRequestIdsValidation
+} = require("../validation/StipendRequestValidation");
+
+/**
+ * @description Create a new stipend request
+ * @route POST /v1/user/request-stipend
+ * @access Private
+ */
+
+exports.requestStipend = catchAsyncError(async (req, res, next) => {
+  const validateData = await validateStipendRequest(req.body);
+
+  const stipend = await StipendRequest.create(validateData.value);
+
+  Mail.sendRecievedStipendRequest(stipend.stipendCategory, stipend.email);
```

Comment:
nit: there's a typo here, should be `sendReceivedStipendRequest`

Reply:
okay I'll correct that. Thanks. 

And @edustipend-root  I'll need email copies. I created a Trello card for it. And also the Heroku account as well.